code/__DEFINES/is_helpers.dm

```diff
@@ -41,6 +41,8 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define isplatingturf(A) (istype(A, /turf/open/floor/plating))
 
+#define isopenspace(A) (istype(A, /turf/open/openspace))
```

Comment:
if https://github.com/tgstation/tgstation/pull/50831/files gets merged earlier this should probably change to an istransparent check

Reply:
Sure, if that get's merged first, then I could change it.  Otherwise, that PR will need to make the change, or I'll have to make another PR in the future.